networkx/algorithms/triads.py

```diff
@@ -245,6 +279,14 @@ def all_triplets(G):
     -------
     triplets : generator of 3-tuples
        Generator of tuples of 3 nodes
+
+    Examples
+    --------
+    >>> G = nx.DiGraph([(1, 2), (2, 3), (3, 4)])
+    >>> triplets = nx.all_triplets(G)
+    >>> print([triplet for triplet in triplets])
+    [(1, 2, 3), (1, 2, 4), (1, 3, 4), (2, 3, 4)]
```

Comment:
I think it'd be slightly more canonical to write this as:

```suggestion
    >>> list(nx.all_triplets(G))
    [(1, 2, 3), (1, 2, 4), (1, 3, 4), (2, 3, 4)]
```